metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```
@@ -262,7 +263,9 @@ def process_dataflow_node(
 
             else:
 
-                raise ValueError(f"Unrecognized Glue data object type: {node_args}")
+                if not self.source_config.ignore_unsupported_connectors:
+
```

probably at least add a log line here so that we are not silently ignoring.